kivy/uix/behaviors/focus.py

```diff
@@ -531,7 +531,7 @@ def keyboard_on_key_down(self, window, keycode, text, modifiers):
         key was consumed.
         '''
         if keycode[1] == 'tab':  # deal with cycle
-            if ['shift'] == modifiers:
+            if 'shift' in modifiers:
```

Comment:
```suggestion
            if {'alt', 'rctrl', 'lctrl', 'meta', 'super', 'alt-gr',
                    'compose'} & modifiers:
                return False

            if {'shift', 'rshift'} & modifiers:
```

E.g. on Windows, if the ctrl or these other special keys are pressed, tab means something else, like switch window (Windows will not even transmit to Kivy these keys when pressed since it uses it). So we shouldn't use them for tabbing, either.

Reply:
`rctrl`, `lctrl`, and `alt-gr` don't show up in `modifiers` on the platforms I have access too, only in `keycode`. I think what we want is
```suggestion
            if {'ctrl', 'alt', 'meta', 'super', 'compose'} & modifiers:
                return false
            if {'shift'} & modifiers:  # or "in"
```

Reply:
I was going by the list from https://github.com/kivy/kivy/blob/master/kivy/core/window/__init__.py#L63. But I guess you're right so feel free to use your suggestion.